functests/config.go

```diff
@@ -0,0 +1,271 @@
+		{
+			APIGroups: []string{"config.openshift.io"},
+			Resources: []string{"images"},
+			Verbs: []string{
+				"get",
+				"list",
+				"watch",
+			},
+		},
+		{
+			APIGroups: []string{""},
+			Resources: []string{
+				"pods",
+				"services",
+				"services/finalizers",
+				"endpoints",
+				"persistentvolumeclaims",
+				"events",
+				"configmaps",
+				"secrets",
+			},
+			Verbs: []string{
+				"create",
+				"delete",
+				"get",
+				"list",
+				"patch",
+				"update",
+				"watch",
+			},
+		},
+		{
+			APIGroups: []string{"apps"},
+			Resources: []string{
+				"deployments",
+				"daemonsets",
+				"replicasets",
+				"statefulsets",
+			},
+			Verbs: []string{
+				"create",
+				"delete",
+				"get",
+				"list",
+				"patch",
+				"update",
+				"watch",
+			},
+		},
+		{
+			APIGroups: []string{"monitoring.coreos.com"},
+			Resources: []string{"servicemonitors"},
+			Verbs: []string{
+				"get",
+				"create",
+			},
+		},
+		{
+			APIGroups:     []string{"apps"},
+			ResourceNames: []string{operatorName},
+			Resources:     []string{"deployments/finalizers"},
+			Verbs:         []string{"update"},
+		},
+		{
+			APIGroups: []string{""},
+			Resources: []string{"pods"},
+			Verbs:     []string{"get"},
+		},
+		{
+			APIGroups: []string{"apps"},
+			Resources: []string{"replicasets", "deployments"},
+			Verbs:     []string{"get"},
+		},
+		{
+			APIGroups: []string{"policy"},
+			Resources: []string{"poddisruptionbudgets"},
+			Verbs: []string{
+				"create",
+				"delete",
+				"get",
+				"list",
+				"patch",
+				"update",
+				"watch",
+			},
+		},
+		{
+			APIGroups: []string{"cincinnati.openshift.io"},
+			Resources: []string{"*"},
+			Verbs: []string{
+				"create",
+				"delete",
+				"get",
+				"list",
+				"patch",
+				"update",
+				"watch",
+			},
+		},
+	},
+}
+
+var clusterRoleBinding = &rbacv1.ClusterRoleBinding{
+	TypeMeta: metav1.TypeMeta{
+		Kind:       "ClusterRoleBinding",
+		APIVersion: "rbac.authorization.k8s.io/v1",
+	},
+	ObjectMeta: metav1.ObjectMeta{
+		Name: operatorName,
+	},
+	Subjects: []rbacv1.Subject{
+		{
+			Kind:      "ServiceAccount",
+			Name:      operatorName,
+			Namespace: operatorNamespace,
+		},
+	},
+	RoleRef: rbacv1.RoleRef{
+		Kind:     "ClusterRole",
+		Name:     operatorName,
+		APIGroup: "rbac.authorization.k8s.io",
+	},
+}
+
+var deployment = &appsv1.Deployment{
+	TypeMeta:   metav1.TypeMeta{Kind: "Deployment", APIVersion: "apps/v1"},
+	ObjectMeta: metav1.ObjectMeta{Name: operatorName},
+	Spec: appsv1.DeploymentSpec{
+		Selector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"name": operatorName,
+			},
+		},
+		Template: v1.PodTemplateSpec{
+			ObjectMeta: metav1.ObjectMeta{
+				Labels: map[string]string{
+					"name": operatorName,
+				},
+			},
+			Spec: v1.PodSpec{
+				ServiceAccountName: operatorName,
+				Containers: []v1.Container{{
+					Name:            operatorName,
+					Image:           "quay.io/cincinnati/cincinnati-operator:latest",
```

Comment:
This should point to the operator image built by the CI, not to quay.

Reply:
Done